src/test/resources/Features/OCD-2916-BasicSurveillanceReporting.feature

```diff
@@ -0,0 +1,118 @@
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2020-Q2  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q2 of year 2020?|
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2019-Q2  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q2 of year 2019?|
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2020-Q3  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q3 of year 2020?|
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2019-Q3  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q3 of year 2019?|
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2020-Q4  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q4 of year 2020?|
+      |ROLE_ADMIN |SLI Compliance   |SLI Compliance-2019-Q4  |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q4 of year 2019?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2020-Q1          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q1 of year 2020?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2019-Q1          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q1 of year 2019?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2020-Q2          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q2 of year 2020?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2019-Q2          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q2 of year 2019?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2020-Q3          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q3 of year 2020?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2019-Q3          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q3 of year 2019?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2020-Q4          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q4 of year 2020?|
+      |ROLE_ACB   |UL LLC           |UL LLC-2019-Q4          |Are you sure you wish to initiate quarterly surveillance reporting for quarter Q4 of year 2019?|
+
+Scenario Outline: User logs in as ROLE and initiates and deletes a quarterly surveillance report for ACB 
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the surveillance reports page
+    And I expand "<ACB>"
+    And I initate a quarterly surveillance report for "<ACB-Year-Quarter>"
+    And I confirm initiaing a quarterly surveillance report
+    Then I see "<Page Title>" for the quarterly surveillance report page
+    And "<Subtitle>" for the quarterly surveillance report 
+    And I delete the quarterly surveillance report
+    Then I see "<Confirmation Message>" on clicking delete
+    And I confirm deleting a quarterly surveillance report
+    Then I see "CHPL Surveillance" as the page title for the surveillance report page
```

Comment:
This should use "Available reports" instead of "CHPL Surveillance". CHPL Surveillance shows on quarterly report page as well, assert won't fail even if delete fails. Also, asset should really be on Q# button label to verify report was deleted than page title, it could be that redirect happened but delete did not succeed/complete